README.md

```diff
@@ -101,3 +100,16 @@ print(workflow.get_result_descriptor())
 
 workflow.get_dataframe(ge.Bbox([-60.0, 5.0, 61.0, 6.0], [time, time]))
 ```
+
+## Authentication
+
+If the Geo Engine server requires authentication, you can set your credentials in the following ways:
+
+1. in the initialize method: `ge.initialize("http://peter.geoengine.io:6060", ("email", "password"))`
```

Comment:
We shouldn't require someone to type in their password in plain text.
Thus, I propose only setting a username and then our library opens an input-prompt that hides the input.